### PR TITLE
Minor text output cleanup for the new account ownership pages

### PR DESCRIPTION
**Pull request by TheAspens**
Pull request #2965 needed some minor adjustments (mostly for text output).  It was easier to make them in a new pull request after merging it rather than go back and forth for minor tweaks.

### CODE REVIEW

File: html/ops/check_account_ownership_keys.php [hunk 1]
```diff
@@ -17,23 +17,25 @@
...
-require_once("../inc/user.inc");
-require_once("../inc/util.inc");
+require_once('../inc/util_ops.inc');
 require_once("../inc/account_ownership.inc");
 
+admin_page_head("Account Ownership Keys");
+
 if (!file_exists($account_ownership_private_key_file_path)) {
-  echo "<p>The '$account_ownership_private_key_file_name' key <b>doesn't</b> exist. Please run the 'generate_account_ownership_keys.php' script from the BOINC web server command line.";
+  echo "<p>The account ownership private key '" . $account_ownership_private_key_file_name . "' <b>doesn't</b> exist. Please run the 'generate_account_ownership_keys.php' script in the BOINC web server ops directory.</p>";
```
Comment (Rytiss) [inline html/ops/check_account_ownership_keys.php]: I think this should still mention that key generation is a command-line task.

File: html/ops/check_account_ownership_keys.php [hunk 2]
```diff
@@ -17,23 +17,25 @@
...
 } else {
-  echo "<p>The '$account_ownership_private_key_file_name' key exists.";
+  echo "<p>The account ownership private key '" . $account_ownership_private_key_file_name . "' exists.</p>";
 }
 
 if (!file_exists($account_ownership_public_key_file_path)) {
-  echo "<p>The '$account_ownership_public_key_file_name' key <b>doesn't</b> exist. Please run the 'generate_account_ownership_keys.php' script from the BOINC web server command line.";
+  echo "<p>The account ownership public key '" . $account_ownership_public_key_file_name . "' <b>doesn't</b> exist. Please run the 'generate_account_ownership_keys.php' script in the BOINC web server ops directory.</p>";
```
Comment (Rytiss) [inline html/ops/check_account_ownership_keys.php]: Same as above.

**Comment by ChristianBeer** [comment]
With the new scripts documentation should be updated here: https://boinc.berkeley.edu/trac/wiki/ProofOfOwnership too.

File: html/ops/check_account_ownership_keys.php (hunk 1 above)
Comment (TheAspens) [reply html/ops/check_account_ownership_keys.php]: updated

File: html/ops/check_account_ownership_keys.php (hunk 2 above)
Comment (TheAspens) [reply html/ops/check_account_ownership_keys.php]: updated

**Comment by TheAspens** [comment]
I've updated the documentation at https://boinc.berkeley.edu/trac/wiki/ProofOfOwnership

**Comment by TheAspens** [comment]
Let me know if there are any other concerns otherwise feel free to merge.

**Pull request merged**